model/converter/json/from_domain.go

```diff
@@ -140,15 +177,15 @@ func (fd fromDomain) convertLogs(logs []model.Log) []json.Log {
 func (fd fromDomain) convertProcesses(processes map[string]*model.Process) map[json.ProcessID]json.Process {
 	out := make(map[json.ProcessID]json.Process)
 	for key, process := range processes {
-		out[json.ProcessID(key)] = fd.convertProcess(process)
+		out[json.ProcessID(key)] = *fd.convertProcess(process)
 	}
 	return out
 }
 
-func (fd fromDomain) convertProcess(process *model.Process) json.Process {
-	return json.Process{
+func (fd fromDomain) convertProcess(process *model.Process) *json.Process {
```

Comment:
this is inefficient, you're always forcing heap allocation. Why not do the reverse, take the pointer only when needed?